js/contributors.js

```diff
@@ -32,7 +32,7 @@ $(document).ready(function(){
   }).done(function(data) {
     shuffle(data)
     data.forEach(function(repos){
-      var html = '<div class="card hvr-hang revealOnScroll single-mentor aos-all aos-item" data-aos="fade-down-right">';
+      var html = '<div class="card hvr-hang revealOnScroll single-mentor aos-all aos-item" data-aos="flip-left">';
```

Comment:
animation was already present. What you added?

Reply:
It was added by me. It was looking messy in **Firefox**, and was **dull** too.
So used another set of animations.